doc/Language/grammar_tutorial.pod6

```diff
@@ -189,21 +160,22 @@ that has all 3 parameters included:
     #  data => ｢7/notify｣
```

Comment:
You see them in the console output when you run the say's was my recollection.
HALFWIDTH LEFT CORNER BRACKET is what a UNICODE search says. I can remove them. They were present in the original version.